ext/fiat/README.md

```diff
@@ -37,7 +37,7 @@ does manage to build that file, but the work on that branch was never finished
 (the correctness proofs of implementation templates still apply, but the
 now abandoned prototype specialization facilities there are unverified).
 
-## Working With Fiat Crypto Field Arithmetic
+## Working with fiat-crypto field arithmetic
```

Comment:
This file comes from an external project and should not be changed, please revert.

Reply:
fixed